extras/bindings/fortran/src/SolverInterfaceFortran.cpp

```diff
@@ -274,6 +274,14 @@ void precicef_get_data_id_(
   *dataID = impl->getDataID(stringDataName, *meshID);
 }
 
+void precicef_isMeshConnectivityRequired(
+    const int *meshID,
+    int *      required)
+{
+  PRECICE_CHECK(impl != nullptr, errormsg);
+  *required = impl->isMeshConnectivityRequired(*meshID);
```

Comment:
```suggestion
  if (impl->isMeshConnectivityRequired(*meshID)) {
    *required = 1;
  } else {
    *required = 0;
  }
```
At least, this is how we do it for similar API functions.